addons/hr_contract/tests/test_employee.py

```diff
@@ -0,0 +1,84 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from datetime import datetime
+
+from .common import TestContractCommon
+
+
+class TestHrEmployee(TestContractCommon):
+
+    @classmethod
+    def setUpClass(cls):
+        super(TestHrEmployee, cls).setUpClass()
+        cls.contracts = cls.env['hr.contract']
+
+    def create_contract(self, state, kanban_state, start, end=None):
+        '''
+        Create a contract with the given state, kanban_state, start date and
+        end date and link it to the test employee.
+
+        :param state (str): the state of the contract
+        :param kanban_state (str): the kanban state of the contract
+        :param start (_Date): the start date of the contract
+        :param end (_Date): the end date of the contract
```

Comment:
I am not sure if this is correct docstring.. and also, this is such a simple function (also it's in test), I am not sure it is needed. So, we could remove it

Reply:
I won't say that's PEP-287 compliant and true it's not really necessary I'll remove it